_config/config.yml

```diff
@@ -49,3 +49,5 @@ After: '#coreauthentication'
 ---
 SilverStripe\Security\RememberLoginHash:
   logout_across_devices: false
+SilverStripe\SessionManager\Models\LoginSession:
+  last_accessed_threshold: 300
```

Comment:
This should be defined in the PHP class instead, if we go ahead with this change.

Reply:
that can be easily fixed if i have your green light with the current approach